econet_base.yaml

```diff
@@ -130,20 +130,6 @@ text_sensor:
     request_mod: 2
     icon: "mdi:alert"
     entity_category: "diagnostic"
-  - platform: econet
-    name: "Alarm History 3"
-    id: alarm_history_3
-    sensor_datapoint: ALARMH03
-    request_mod: 3
-    icon: "mdi:alert"
-    entity_category: "diagnostic"
-  - platform: econet
-    name: "Alarm History 4"
-    id: alarm_history_4
-    sensor_datapoint: ALARMH04
-    request_mod: 3
-    icon: "mdi:alert"
-    entity_category: "diagnostic"
```

Comment:
Are these useful to leave in should someone want them? E.g. comment them out or set the request_mod polling interval to none?

Reply:
I don't see how these would be useful given HA has its own history anyway. I'd rather not leave commented out code. It's already easy for someone to copy the existing alarm history sensors and modify them to include more. And it's much easier to call the read service from HA.

Reply:
If you just click on the sensor you will get the history, e.g. for a different string sensor:
![image](https://github.com/esphome-econet/esphome-econet/assets/9987465/fb86a22d-1fd5-43cd-8c7b-fe1d37eff590)